ghwatch.py

```diff
@@ -141,6 +148,8 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument('--days', '-d', type=int, default=7, help='Number of days to look back (default: 7)')
     parser.add_argument('--refresh-time', '-r', type=int, default=600, help='Refresh time in seconds in interactive mode (default: 600)')
     parser.add_argument('--default-config', action='store_const', const=True, default=False, help='Generate a default configuration file in ~/.config/ghwatch')
+    parser.add_argument('--sort',  action='store_true', default=None, help="Sort notifications by reasons (and then time). Overrides 'sort_notifications' in the configuration file")
+    parser.add_argument('--no-sort', dest='sort', action='store_false', help="Don't sort notifications. Overrides 'sort_notifications' in the configuration file")
```

Comment:
I think it would be handy to have short versions of these args... `-s` and `-ns`?

I'd be tempted to propose adding args to sort only by time. Not for this pull, but mentioning because I'm not sure what arg names would be good in that case.

Reply:
i'm not sure `-ns` would work, can short options be multi char ? (in GNU syntax that would mean `-n -s`)

Reply:
oops, good point